en/reference/annotations-mapping.rst

```diff
@@ -49,6 +49,10 @@ Document
 Optional attributes:
 
 -  **nodeType**: PHPCR type for this node, default ``nt:unstructured``.
+-  **uniqueNodeType**: If this document uses a unique node type, set to ``true``
```

Comment:
maybe "If this document has a unique node type, set to ..."? and mention that when this is set, you need to be sure to assign a custom node type for each document class (and link to the documentation part about node type registering)

also, we should mention the command to verify that they node types really are unique.